helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -263,7 +262,7 @@ private Response batchGetStoppableInstances(String clusterId, JsonNode node, boo
         break;
       case instance_based:
       default:
-        throw new NotImplementedException("instance_based selection is not supported yet!");
+        throw new UnsupportedOperationException();
```

Comment:
Could you please keep the error message here?

Reply:
My apologies.  That was a silly omission by me.  Thank you for catching it.